### ISSUE TITLE
Questions on the code `acg_node_localization.cpp`

### ISSUE THREAD

**Issue by ljjTYJR** [MalcolmMielle/Auto-Complete-Graph#13]
In the line 

https://github.com/MalcolmMielle/Auto-Complete-Graph/blob/ffe7b0df929d8a341fc595d1391d6a623e7f86c3/src/Localization/acg_node_localization.cpp#L890

Where does the param come from? `g2o::SE2(0.2, 0.1, -0.1)`, it seems the offset of the sensor?

**Comment by MalcolmMielle** [comment]
Yep after looking in the code, I belive it's an offset used in the graph optimization in g2o:

https://github.com/MalcolmMielle/Auto-Complete-Graph/blob/ffe7b0df929d8a341fc595d1391d6a623e7f86c3/include/auto_complete_graph/OptimizableAutoCompleteGraph.hpp#L50-L54
